symfony/plugins/orangehrmAdminPlugin/config/routes.yaml

```diff
@@ -110,6 +110,20 @@ apiv2_licenses:
   defaults:
     _api: OrangeHRM\Admin\Api\LicenseAPI
 
+apiv2_organization:
```

Comment:
can you try, it should only expose `getOne` and `update`
```yaml
apiv2_organization:
  path: /api/v2/admin/organization
  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
  methods: [ GET, PUT ]
  defaults:
    id: 1
    _api: OrangeHRM\Admin\Api\OrganizationAPI
```

Reply:
changed. But path is    /api/v2/admin/organization/{id}